v2/internal/genericarmclient/generic_client_test.go

```diff
@@ -12,13 +12,16 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	. "github.com/onsi/gomega"
+
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1api20200601"
```

Comment:
Where is this actually used now? As far as I can tell there isn't any actual usage of this import here.

Reply:
Used in the test `Test_NewResourceGroup`, see line 63.

Reply:
Find in IE betrayed me.